lib/views/addToCollection.js

```diff
@@ -0,0 +1,119 @@
+const loadTemplate = require('../loadTemplate')
+const pug = require('pug')
+const config = require('../config')
+const sparql = require('../sparql/sparql')
+const getOwnedBy = require('../query/ownedBy')
+
+module.exports = function (req, res) {
+  req.setTimeout(0) // no timeout
+
+  function addToCollectionsForm (req, res) {
+    var collectionQuery = 'PREFIX dcterms: <http://purl.org/dc/terms/> PREFIX sbol2: <http://sbols.org/v2#> SELECT ?object ?name WHERE { ?object a sbol2:Collection . OPTIONAL { ?object dcterms:title ?name . }}'
```

Comment:
Can we put this in a .sparql file?

Reply:
Actually it is there (getCollections.sparql).  Will switch to use it.